backend/populators/tickers.js

```diff
@@ -134,4 +135,100 @@ const companyFillHelper = async (cik) => {
   }
 };
 
+// assign sector and industry denominations to database
+router.post("/sectorfill", async (req, res) => {
+  const companies = await prisma.company.findMany();
+  for (let company of companies) {
+    const companyInfo = await yahooFinance.quoteSummary(company.ticker, {
+      modules: ["assetProfile"],
+    });
+    console.log(companyInfo);
+    res.json(companyInfo);
+    await prisma.industry.create({
+      name: companyInfo.industry,
+    });
```

Comment:
would this result in duplicated industry?

Reply:
should we also save `companyInfo.industry` and other metadata into `prisma.company`?